koiki/woocommerce/wcfmmp.py

```diff
@@ -15,4 +15,7 @@ def request(self, path):
         abs_url = f'{self.api_url}/{path}'
         self.logger.info(f'Wcfmpp request. url={abs_url}')
 
-        return self.client.get(abs_url)
+        response = self.client.get(abs_url, auth=(koiki.wcfmmp_user, koiki.wcfmmp_password))
```

Comment:
From what I see in most libraries or helpers, normally the auth is done once with a 'auth' method of the client and it can be saved as internal class variables, ex: self.user and self.password  to be used later in the get method or other methods like update, etc... that way no need to send the auth everytime.

Reply:
yes, I think user and password do feel like this class's internal state.